wpilibc/src/main/native/cpp/logging/LogCell.cpp

```diff
@@ -20,3 +27,15 @@ void LogCell::ClearCell() { m_content.clear(); }
 void LogCell::AcquireLock() { m_mutex.lock(); }
 
 void LogCell::ReleaseLock() { m_mutex.unlock(); }
+
+bool LogCell::IsStringValid(std::string string) {
+  if (string.find('\"') != std::string::npos) {
+    std::cout << "LogCell's name/content cannot contain double quotes"
```

Comment:
This seems unnecessarily prohibitive.

Reply:
So what should I do then ?

Reply:
Escape the quotes?

I concur with @calcmogul on having a general log file class instead of badly doing CSV though.